src/main/java/uk/gov/hmcts/reform/ucmc/handler/ConfirmServiceCallbackHandler.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.reform.ucmc.handler;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.ucmc.callback.Callback;
+import uk.gov.hmcts.reform.ucmc.callback.CallbackHandler;
+import uk.gov.hmcts.reform.ucmc.callback.CallbackParams;
+import uk.gov.hmcts.reform.ucmc.callback.CallbackType;
+import uk.gov.hmcts.reform.ucmc.callback.CaseEvent;
+import uk.gov.hmcts.reform.ucmc.enums.ServiceMethod;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.ucmc.callback.CaseEvent.CONFIRM_SERVICE;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.DATE;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.DATE_TIME_AT;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.formatLocalDate;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.formatLocalDateTime;
+
+@Service
+public class ConfirmServiceCallbackHandler extends CallbackHandler {
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(CONFIRM_SERVICE);
+
+    private final ObjectMapper mapper;
+
+    public ConfirmServiceCallbackHandler(ObjectMapper mapper) {
```

Comment:
can we use lombok annotation for it?

Reply:
I seem to remember an issue with lombok and constructors with one parameter, but sure I will try swapping!